api.docs/bruno/glific_api/Contacts/Other filters on Contacts.bru

```diff
@@ -43,3 +43,64 @@ body:graphql:vars {
     }
   }
 }
+
+docs {
+  # Other Filters on Contacts
+  
+  ## Description
+  This GraphQL API allows you to apply additional filters to retrieve contacts based on specific criteria, such as including certain groups or tags.
+  
+  ## Base URL
+  `{{api_url}}`
+  
+  ## URL
+  (No specific URL for GraphQL, as it is usually handled by a single endpoint.)
+  
+  ## Parameters
+  
+  | Parameter | Type           | Description                                                              |
+  |-----------|----------------|--------------------------------------------------------------------------|
+  | `filter`  | ContactFilter  | Filters to apply for contact retrieval.                                  |
+  | `opts`    | Opts           | Options for ordering, limiting, and offsetting results.                  |
```

Comment:
Add all possible values of filter and opts in tabular form along with description